Easy/leftRotation.rb

```diff
@@ -0,0 +1,12 @@
+#A left rotation operation on an array of size n shifts each of the array's elements 1 unit to the left. Given an integer, d , rotate the array that many steps left and return the result.
+def rotateLeft(number_of_left_rotations, input_array)
+    # Write your code here 
+    # arr.rotate(d)
+    for i in 0...number_of_left_rotations
+        shifting_element = input_array.shift
+        input_array << shifting_element
```

Comment:
Also can you think of writing this with Array#split, reverse and join

Reply:
Used .times
Used .reverse and .pop to achieve the same result
Also got to know that -> .rotate(number_of_times) helps achieving the same result